### PR TITLE
Fixed projector links in autopilot- and projectors-views

### LINKED ISSUE #1041: Link from autopilot to a projector is broken

### LINKED ISSUE #1042: A delegate cannot see a projector in fullscreen

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #1041 
Closes #1042 
Found out that both the links in the autopilot component and in the projector-list-entry component were broken in different ways and proceeded to fix them all.

### CODE REVIEW

**Review by GabrielInTheWorld — APPROVED** [review]

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Open projector works now from autopilot.
But if you go to main menu entry "Projector" and opens (as delegate user) the projector by click you get a black page...

**Review by emanuelschuetze — APPROVED** [review]

File: client/src/app/site/projector/components/projector-list-entry/projector-list-entry.component.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class ProjectorListEntryComponent extends BaseComponent {
         if (this.operator.hasPerms(Permission.projectorCanManage)) {
             return `/${this.activeMeetingId}/projectors/detail/${this.projector.sequential_number}`;
         } else {
-            return `/${this.activeMeetingId}/projector/${this.projector.sequential_number}`;
+            return `/${this.activeMeetingId}/projector/${this.projector.id}`;
```
Comment (GabrielInTheWorld) [inline client/src/app/site/projector/components/projector-list-entry/projector-list-entry.component.ts:90]: Will this still work? Please check!

**Review by GabrielInTheWorld — CHANGES_REQUESTED** [review]

**Comment by luisa-beerboom** [comment]
Currently not functional:
Links should open a new tab with the fullscreen projector, but instead the new tab contains the calendar with the url:
https://localhost:8000/?error=Authentication%20Error&msg=meeting.can_see_frontpage

Probably related to issue #1080

**Comment by GabrielInTheWorld** [comment]
> Currently not functional: Links should open a new tab with the fullscreen projector, but instead the new tab contains the calendar with the url: https://localhost:8000/?error=Authentication%20Error&msg=meeting.can_see_frontpage
> 
> Probably related to issue #1080

A projector should be requested by its `sequential_number`. Please try this and give me a feedback, if its still not work.

**Comment by luisa-beerboom** [comment]
> > Currently not functional: Links should open a new tab with the fullscreen projector, but instead the new tab contains the calendar with the url: https://localhost:8000/?error=Authentication%20Error&msg=meeting.can_see_frontpage
> > Probably related to issue #1080
> 
> A projector should be requested by its `sequential_number`. Please try this and give me a feedback, if its still not work.

The reason I currently request it by it's id, instead of the sequential number, is bc these were the correct and working routes to the fullscreen-page before the rebase to the new version. Unless the Angular 13 update changed the routes, they therefore still should be correct.
Just for the sake of being safe I _did_ try out the `sequential_number`-routes too, those also didn't work and, as far as I remember, instead routed to the default route, not 100% sure on that, I will try it again later.

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**